src/raft/raft.rs

```diff
@@ -664,6 +668,12 @@ impl<T: Storage> Raft<T> {
             if self.state == StateRole::Leader {
                 debug!("{} ignoring MsgHup because already leader", self.tag);
             } else {
+                let ents = self.raft_log.entries(self.raft_log.applied + 1, self.raft_log.committed - self.raft_log.applied).expect("unexpected error getting uncommitted entries");
```

Comment:
This is added [here](https://github.com/coreos/etcd/pull/6039/files#diff-a8629a0eb6da1c6c3805111f1f1af85eR577)